Lib/test/test_unicodedata.py

```diff
@@ -216,6 +216,12 @@ def test_issue29456(self):
         self.assertEqual(self.db.normalize('NFC', u11a7_str_a), u11a7_str_b)
         self.assertEqual(self.db.normalize('NFC', u11c3_str_a), u11c3_str_b)
 
+    def test_is_normalized_unicode_3_2_0(self):
```

Comment:
How slow is this test? If it takes a significant amount of time (in comparison with other tests), perhaps we only need to test few example cases.

I would test cases which return different results for 3.2.0 and the current database. Also, is it worth to include tests for multicharacter strings? Maybe take some examples from other tests for `normalize()` and `is_normalized()`.

Reply:
> I would test cases which return different results for 3.2.0 and the current database. Also, is it worth to include tests > for multicharacter strings? Maybe take some examples from other tests for normalize() and is_normalized().

Okay, I will try it!